src/Directory.Build.props

```diff
@@ -1,22 +1,15 @@
 <!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
 <Project>
   <Import Project="..\Directory.Build.props"/>
-
-  <!-- This file is imported by all projects at the beginning of the project files -->
   <Import Project="$(RepoToolsetDir)Settings.props" />
 
   <PropertyGroup>
-    <!-- Enable IOperation feature for IOperation based analyzers. -->
```

Comment:
This still needs to be retained. I am wondering how the build did not fail without this flag...